### DISCUSSION TITLE
macos 12 Monterey jobs don't start

### DISCUSSION THREAD

**Discussion by gereons** [actions/virtual-environments#5344] [General questions] [answered]
We have a bunch of jobs that should run on macos-12 VMs (and did so in the past few days, basically since we were allowed access to the beta program).

Since yesterday, we noticed that our jobs are queued, but never run. This happens both in private and in public repos.
Anything we can do about this?

The complete action output is

> Requested labels: macos-12
Job defined at: snabble/iOS-SDK/.github/workflows/lint.yml@refs/pull/88/merge
Waiting for a runner to pick up this job...

**Comment by miketimofeev** [comment, marked as answer]
@gereons something definitely went wrong, we are investigating the issue at the moment

**Reply by gereons** [reply]
Jobs are working again, thanks for the fix!

**Comment by vitorll** [comment]
We are still seeing this in our workspace in more than one branch, what could it be?

```
[verify_pull_request](https://github.com/actions/virtual-environments/discussions/5344#logs)
Started 9m 28s ago
Requested labels: macos-12
Job defined at: .github/workflows/verify_pull_request.yml@refs/pull/1780/merge
Waiting for a runner to pick up this job...
```

**Reply by miketimofeev** [reply]
@vitorll could you share the links and what is the org name?

**Reply by vitorll** [reply]
hey @miketimofeev is there a way to share that privately?

**Comment by usersxx** [comment]
This problem has been seen in [macports/macports-ci-files](https://github.com/macports/macports-ci-files) repository [where macos-12 workflow hasn't started](https://github.com/macports/macports-ci-files/actions/runs/2083971638)

The total duration was 1d 14h 43m 11s

**Reply by miketimofeev** [reply]
Are the new ones also stuck?

**Reply by usersxx** [reply]
Sadly, the new ones are also stuck. (see: https://github.com/macports/macports-ci-files/runs/5887687757?check_suite_focus=true)

**Reply by miketimofeev** [reply]
@usersxx have you submitted the form to access the private preview?  https://forms.gle/D1ihWjw3xDfjYbu67

**Reply by usersxx** [reply]
> @usersxx have you submitted the form to access the private preview? https://forms.gle/D1ihWjw3xDfjYbu67

No

**Reply by usersxx** [reply]
> > @usersxx have you submitted the form to access the private preview? https://forms.gle/D1ihWjw3xDfjYbu67
> 
> No

I didn't know about it

**Reply by usersxx** [reply]
So now I know why the workflows are stuck.

**Reply by chrisxiao** [reply]
Hi @miketimofeev , I have the same problem, I submited the form above, but not works. 
see: https://github.com/chrisxiao/TrojanA/actions/runs/2193851207

**Reply by miketimofeev** [reply]
@chrisxiao we onboard users weekly, the next turn will be this Friday. After the access is granted we send welcome e-mail so make sure to check your inbox

**Reply by chrisxiao** [reply]
@miketimofeev  ok, thanks.

**Comment by samus-aran** [comment]
We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
